packages/dashboard/test/helpers.ts

```diff
@@ -374,3 +375,14 @@ function onceInvoked(object, functionName): Promise<void> {
 export async function onceResized(dashboard: HTMLElement): Promise<void> {
   await onceInvoked(dashboard, '_onResize');
 }
+
+export async function updateComplete(dashboard: HTMLElement): Promise<void> {
+  await nextUpdate(dashboard);
+
+  const widgetsAndSections = dashboard.querySelectorAll('vaadin-dashboard-widget, vaadin-dashboard-section');
+  for (const child of widgetsAndSections) {
+    await nextUpdate(child as HTMLElement);
+  }
+
+  await nextFrame();
```

Comment:
Is `nextFrame` still needed now? If yes, I'd suggest adding a comment what it specifically awaits.

Reply:
Added the comment and an additional `aTimeout(0)` as suggested